fec/fec/static/js/calendar.js

```diff
@@ -47,27 +85,48 @@ Calendar.prototype.updateLinks = function(params) {
   };
   this.$download.html(templates.download(urls));
   this.$subscribe.html(templates.subscribe(urls));
+
+  new dropdown.Dropdown(this.$download, {checkboxes: false});
+  new dropdown.Dropdown(this.$subscribe, {checkboxes: false});
 };
 
-var colorMap = {
-  election: 'blue',
-  report: 'red'
+Calendar.prototype.styleButtons = function() {
+  var baseClasses = 'button button--neutral';
+  this.$calendar.find('.fc-button').addClass(baseClasses);
+  this.$calendar.find('.fc-next-button').addClass('button--next');
+  this.$calendar.find('.fc-prev-button').addClass('button--previous');
+  this.$calendar.find('.fc-right .fc-button-group').addClass('toggles--buttons');
 };
-var colorDefault = 'orange';
 
-function formatColor(event) {
-  var category = event.category ? event.category.split('-')[0] : null;
-  return colorMap[category] || colorDefault;
+var classMap = {
+  election: 'fc--election',
+  report: 'fc--deadline',
+  open: 'fc--meeting',
+  executive: 'fc--executive',
+  roundtables: 'fc--outreach',
+  conferences: 'fc--outreach',
+  litigation: 'fc--other',
+  fea: 'fc--other'
+};
+
+function setClass(event) {
```

Comment:
I would expect a function that starts with `set` to change a value, which this doesn't. How about something more descriptive, like `getEventClass`?